octoprint_mrbeam/templates/tab_workingarea.jinja2

```diff
@@ -50,7 +50,7 @@
 
 													<div class="row-fluid print-control btn-group" style="display: none;" data-bind="visible: loginState.isUser">
 														<button id="laser_button" class="btn btn-danger span6"
-                                                                data-bind="click: conversion.show_conversion_dialog, enable: state.isOperational() && state.isReady() && !state.isPrinting() && !state.isPaused() && loginState.isUser() && !workingArea.working_area_empty(), attr: {title: state.titlePrintButton}"
+                                                                data-bind="click: conversion.show_conversion_dialog, enable: {{ 'true' if isDevelop and 'enableLaserButton' in request.args else 'false'}} || (state.isOperational() && state.isReady() && !state.isPrinting() && !state.isPaused() && loginState.isUser() && !workingArea.working_area_empty()), attr: {title: state.titlePrintButton}"
```

Comment:
why you add this additional 'true' and 'false
it would be easier with
```suggestion
                                                                data-bind="click: conversion.show_conversion_dialog, enable: (isDevelop && 'enableLaserButton' in request.args) || (state.isOperational() && state.isReady() && !state.isPrinting() && !state.isPaused() && loginState.isUser() && !workingArea.working_area_empty()), attr: {title: state.titlePrintButton}"
```

Reply:
Oh I didn't know that is possible 😅. This is needed to access the args right?
But do we still need to expose the isDevelop?